pydov/types/monster.py

```diff
@@ -7,6 +7,121 @@
 import datetime
 import numpy as np
 
+class TijdstipField(_CustomXmlField):
+    """Field for retrieving the time from the relevant XML
+    field."""
+
+    def __init__(self, name, definition, source_xpath):
```

Comment:
I would call this `base_xpath` instead of `source_xpath` to indicate this is not the full XPath expression.

Can you also add a docstring to this function?